doc/hardware/sensors.rst

```diff
@@ -0,0 +1,41 @@
+
+Sensors list
+##################################
+
+.. table:: Supported sensors on MinnowBoard MAX, Galileo GEN 2 and Edison boards
+
+    ============== ============================================= ==========================================================================
+    Sensor Name     Sensor Type                                    Comment
+    ============== ============================================= ==========================================================================
+    BMA180          Accel                                          missing driver on Edison
+    BMA250          Accel                                          missing driver on Edison
+    LIS3DH          Accel                                          missing driver on Edison
+    MMA7455         Accel                                          missing driver on Edison
+    BMA255          Accel                                          missing driver on Edison
+    LIS2DH12TR      Accel                                          missing driver on Edison
+    BMA222E         Accel                                          missing driver on Edison
+    LSM330DLC       Gyro
+    MPU6050         Gyro
+    MPU9250         Gyro
+    L3G4200         Gyro
+    LSM303          Magnet
```

Comment:
Since you're sorting this table by "sensor type", a better presentation might be to pull out that column and use it as a section heading and have separate tables for each type.  That would also let you have better sensor type descriptions. For example, "Magnet" should really be "Triple Axis Accelerometer and Magnetometer (Compass)".